Controller/Redirect/Process.php

```diff
@@ -353,6 +355,15 @@ public function execute()
     protected function handleFailed($statusCode)
     {
         $this->logger->addDebug(__METHOD__ . '|7|');
+
+        $this->eventManager->dispatch('buckaroo_process_handle_failed_before');
```

Comment:
does this require an event - or could have been handled via plugins (and check visibility of the method).

Reply:
yes it`s require event, on next step will be used in separate module